### PR TITLE
Update live-tracing.md

### PR DESCRIPTION
**Pull request by MariusVanDerWijden**
We need to specify which tracer to run

### CODE REVIEW

**Review by s1na — APPROVED** [review]
LGTM

File: docs/developers/evm-tracing/live-tracing.md [hunk 1]
```diff
@@ -292,7 +292,7 @@ Note-worthy explanations:
 First compile the source by running `make geth`. Then run the following command:
 
 ```bash
-./build/bin/geth --vmtrace --vmtrace.config '{"config": "supply-logs"}' [OTHER_GETH_FLAGS]
```
Comment (s1na) [inline docs/developers/evm-tracing/live-tracing.md:295]: Ah wait, I think we also renamed the config flag. Was this working for you?

Comment (MariusVanDerWijden) [reply docs/developers/evm-tracing/live-tracing.md:295]: I didn't use a config, so I removed the config flag when trying my tracer

Comment (s1na) [reply docs/developers/evm-tracing/live-tracing.md:295]: Yep it was renamed to `--vmtrace.jsonconfig`. Can you please fix that too?
https://github.com/ethereum/go-ethereum/blob/86a1f0c39494c8f5caddf6bd9fbddd4bdfa944fd/cmd/utils/flags.go#L548

Comment (MariusVanDerWijden) [reply docs/developers/evm-tracing/live-tracing.md:295]: done

**Review by s1na — APPROVED** [review]
Thanks

**Pull request merged**